cinema/migrations/0002_initial.py

```diff
@@ -18,17 +18,23 @@
         migrations.AddField(
             model_name="order",
             name="user",
-            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL),
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL
+            ),
         ),
         migrations.AddField(
             model_name="moviesession",
             name="cinema_hall",
-            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to="cinema.cinemahall"),
+            field=models.ForeignKey(
+                on_delete=django.db.models.deletion.CASCADE, to="cinema.cinemahall"
```

Comment:
Verify that the `CinemaHall` model exists in the `cinema` app and that the foreign key relationship is correctly set up.